code/obj/item/device/pda2/pda2.dm

```diff
@@ -5,7 +5,7 @@
 	desc = "A portable microcomputer by Thinktronic Systems, LTD. It has a slot for an ID card, and a hole to put a pen into."
 	icon = 'icons/obj/items/pda.dmi'
 	icon_state = "pda"
-	item_state = "pda"
+	//item_state = "pda" NO, STOP FIX ME SPRITING
```

Comment:
```suggestion
```
idk what the comment means but I'm assuming this is unnecessary or broken and thus Delete

Reply:
whoops, it actually does need that. the reason it was commented out was that it was trying to use the inhand iconstate due to belt overlay code using it previously